src/Autocomplete/src/Controller/EntityAutocompleteController.php

```diff
@@ -38,6 +42,11 @@ public function __invoke(string $alias, Request $request): Response
             throw new NotFoundHttpException(sprintf('No autocompleter found for "%s". Available autocompleters are: (%s)', $alias, implode(', ', $this->autocompleteFieldRegistry->getAutocompleterNames())));
         }
 
+        $extraOptions = $this->getExtraOptions($request);
+        $autocompleter = $autocompleter->withOptions([
```

Comment:
I have this vision where "withOptions" means immutable and a new object.. If this is not the goal here, could we use a more usual "setOptions" or "addOptions" ?

Reply:
`setOptions` sounds cool 🤔. Yeah, `with` might create some confusion here.